run_wfdiff_test.py

```diff
@@ -0,0 +1,104 @@
+# To run this script
```

Comment:
I feel like this should also not be part of the repository or at least it should be move to some `/examples` subfolder.